crates/agglayer/src/main.rs

```diff
@@ -11,7 +11,15 @@ fn main() -> anyhow::Result<()> {
     match cli.cmd {
         cli::Commands::Run { cfg } => agglayer_node::main(cfg)?,
         cli::Commands::Prover { cfg } => agglayer_prover::main(cfg)?,
-        cli::Commands::Config { base_dir } => println!(
+        cli::Commands::Config { prover: true, .. } => println!(
```

Comment:
Since this ignores the `base_dir` value, it would probably be cleaner if it was a separate subcommand, i.e. we'd have something like `generate-node-config` and `generate-prover-config` each accepting switches relevant to given mode.

Reply:
Also, a snapshot test for default prover config would be nice, similar to what we have for node config.

Reply:
Yeah I didn't wanted to have too much changes on that but it makes sense 👌🏼